<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TNGAI4U4I</class>
 <widget class="QWidget" name="TNGAI4U4I">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ch 3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Ch 0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Ch 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ch 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ch 2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ch 0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Ch 2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Ch 3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Voltage</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_voltage_ch0">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_voltage_ch1">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_voltage_ch2">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_voltage_ch3">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_current_ch0">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_current_ch1">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="label_current_ch2">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QLabel" name="label_current_ch3">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" colspan="4">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Current</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
